code/SINGE_GLG_Test.m

```diff
@@ -9,18 +9,23 @@ function SINGE_GLG_Test(Data,varargin)
 params = parseParams(Data,varargin{:})
 mkdir(params.outdir)
 %%% End InputParser
-tic;
 if isdeployed
     set(0,'DefaultFigureVisible','off');
 end
-resampling_method = {'holes';'burst'};
-
 
 % Use a temporary matfile to track intermediate state
 % Create new file for each job ID to avoid bugs when storage is shared
 % between parallel jobs
 m = matfile(['TempMat' '_' num2str(params.ID)],'Writable',true);
 
+if ismember('branches',who(m))
+    params.branching = 1;
+    branches = m.branches;
+    n_branches = min(size(branches));
```

Comment:
Do we want n_branches to be the number of unique positive integer values in the current implementation?  What happens if the values are all 0, 1, or 3?

More generally, I'm wondering about the structure of the branches matrix.  We could have each column _i_ correspond to indicator variables that determine whether that cell is part of branch _i_.  Do we need to have integer-valued entries or could binary variables be equivalent?